onnxruntime/core/providers/webnn/builders/model_builder.cc

```diff
@@ -84,16 +84,18 @@ void ModelBuilder::PreprocessActivations() {
     const auto* node(graph_viewer_.GetNode(node_indices[i]));
     const auto& op_type(node->OpType());
 
+    // Ignore unsupported activation nodes.
+    if (!Contains(GetSupportedActivationNodes(), op_type)) {
```

Comment:
It appears this inner loop is creating a new string hashset every single time. Can we just get `GetSupportedActivationNodes` once outside the loop? (or maybe just cache it in the ModelBuilder_ instance, since it's also queried from other places)

Reply:
Good point, thanks!

I've added a `supported_activation_nodes_` private Set instead, PTAL again!